### ISSUE TITLE
SimulationCraft 7.1.5 Crashing whenever I try to Sim

### ISSUE THREAD

**Issue by Lilacs328** [simulationcraft/simc#3571]
Hey I just downloaded SimulationCraft and every time I try to sim my computer crashes. I'm using Windows 10 64-bit and the 64-bit version of simcraft 7.1.5. Including my DxDiag and simc_gui.simc files.

[DxDiag.txt](https://github.com/simulationcraft/simc/files/792670/DxDiag.txt)

https://www.dropbox.com/s/lvyetw0hnrkute9/simc_gui.simc?dl=0

There we go

**Comment by Lilacs328** [comment]
https://www.dropbox.com/s/lvyetw0hnrkute9/simc_gui.simc?dl=0

There we go

**Comment by Lilacs328** [comment]
My computer simply shuts down I don't get any "Program not responding" errors it just turns off. And this happens with any simulation attempt, from sample profiles to real characters.

**Comment by Lilacs328** [comment]
Well I think I figured out the problem I'm having some huge overheating issues. According to SpeedFan my core temp idles around 38C and as soon as I start simming it jumps to 65C+. I didn't have SpeedFan when I first tried the sims but I'm assuming the temperature continues rising as the sim progresses, and since installing SpeedFan I've been monitoring it and pausing the sim every time it rises above 60C until it drops back down. So far I'm getting further along with the sim without the crashing.I'm going to reseat my heatsink with some more thermal paste tomorrow. Can anyone give me a suggestion for some good Chassis Fans because the ones I'm currently using can't seem to get above 600 RPM so I'm sure that's also a contributing factor.
